lib/storage.js

```diff
@@ -1,18 +1,32 @@
 const raf = require('random-access-file')
 const Corestore = require('corestore')
 const join = require('path').join
+const fsctl = require('fsctl')
```

Comment:
we need to make this browser friendly also, load it in a try catch and make sure to only use it if set below, ie

```
let fsctl = null
try {
  fsctl = require('fsctl')
} catch {
  // not supported, ie browser or unsupported platform - just skip
}